lib/adapter.js

```diff
@@ -153,6 +153,13 @@ function doNextCompaction(self) {
   });
 }
 
+function attachmentNameError(name) {
+  if (name.charAt(0) === '_') {
+    return 'Attachment name can\'t start with \'_\'';
```

Comment:
Yeh will probably will do that on the way in, I was doing compatibility with Couch but them we avoid testing for exact error message so itll be fine